### ISSUE TITLE
Mk 1-2 Pod Umbilical Port toggleable outside of the VAB/SPH editor scenes 

### ISSUE THREAD

**Issue by sumghai** [sumghai/SDHI_ServiceModuleSystem#63]
v2.0 introduced a toggleable service module umbilical port model that can be optionally retrofitted to the stock Mk1-2 Command Pod (#3). At the time, it used FSanimateGeneric from the Firespitter plugin, which was configured such that the umbilical port can only be toggled while in the VAB/SPH editor scene (and not the flight or EVA scene).

In v2.3, Firespitter was dropped as a dependency in favor of the stock KSP ModuleAnimateGeneric (#27), which at the time did not distinguish between different scenes, resulting in the umbilical port now being toggleable while in-flight.

KSP 1.0.5 features a new update to ModuleAnimateGeneric, which now supports additional fields with the following defaults:

```
actionAvailable = true
eventAvailableEditor = true
eventAvailableFlight = true
eventAvailableEVA = true
evaDistance = 5
```

By setting **eventAvailableFlight** and **eventAvailableEVA** to **false**, it may be possible to bring back the "VAB/SPH editor-only" behaviour from Firespitter, while still using the stock animation module.

**Comment by sumghai** [comment]
Adding said parameters does not seem to work in KSP 1.0.5. 

Investigation continues.

**Comment by sumghai** [comment]
@NathanKell, I know that you're the one who implemented the scene-specific animation toggling feature, but since you're (understandably) busy with fixes and QA for KSP 1.1, I think it may be best to wait until 1.1 drops and you've had a chance to rest, before we revisit this.

**Comment by sumghai** [comment]
Just spoke to NathanKell via IRC, and apparently, the variables were misspelt as **Avialable**.

Will need to test this tonight.

**Comment by NathanKell** [comment]
Yep. actionAvailable is spelled correctly; the other three are eventAvialableXXX

**Comment by sumghai** [comment]
Fixes tested and working, ready for v3.1.
